stable/app/templates/cron.yaml

```
@@ -36,6 +36,9 @@ spec:
               imagePullPolicy: {{ $.Values.image.pullPolicy }}
               command:
               {{- toYaml $job.command | nindent 16 }}
+              {{- if .Values.otelcollector.enabled }}
+                  pkill -SIGTERM otelcol-contrib
+              {{- end }}
```

no need this, we can override via values
```suggestion
              {{- if .Values.otelcollector.enabled }}
                  pkill -SIGTERM otelcol-contrib
              {{- end }}
```